packages/storybook/stories/CallComposite/JoinExistingCallWithCTE.stories.tsx

```diff
@@ -33,6 +36,9 @@ const JoinExistingCallAsTeamsUserStory = (args: ArgsFrom<typeof storyControls>,
           fluentTheme={context.theme}
           rtl={context.globals.rtl === 'rtl'}
           locator={args.callLocator}
+          meetingLink={args.callLocator}
+          meetingId={args.teamsMeetingId}
```

Comment:
lets see if we can split up this preview so we dont have meetingLink + id/passcode on the same preview or else it will be confusing

Reply:
Yes, thanks. I've slit them to separate page